src/routes/_layout.svelte

```diff
@@ -1,24 +1,38 @@
 <script>
-	import {mq} from 'app/stores/mq';
+	import {onMount} from 'svelte';
+	import {fontScaling} from 'app/stores/fontScaling';
 
 	import Nav from 'app/components/Nav.svelte';
+	import ScreenGauge, {screenGauge} from 'app/components/ScreenGauge.svelte';
+
+	const dev = process.env.NODE_ENV === 'development';
 
 	export let segment;
+	let rootStyle;
+
+	onMount(() => {
+		rootStyle = document.documentElement.style;
+	})
+
+	// set document root element font size so that `rem` units work
+	$: rootStyle && ( rootStyle.fontSize = `${$fontScaling * 16}px`);
```

Comment:
we have discussed about:
- making this portion into an utility to import here
- measuring the root font size on mount